compiler/rustc_borrowck/src/diagnostics/mod.rs

```diff
@@ -586,13 +578,15 @@ impl UseSpans<'_> {
         }
     }
 
+    //FIXME: nested migration case
```

Comment:
```suggestion
    // FIXME(#100717) requires eager translation/list support
```
similarly for others